CHANGELOG/1.2-TEAM.md

```diff
@@ -0,0 +1,12 @@
+# v1.2 Release Team
+
+| Enhancement | Leads |
+| - | - |
+| Infrastructure Labels and Annotations | @keithmattix, @snorwin |
+| HTTPRoute Timeouts and Durations | @kflynn, @xtineskim |
+| BackendProtocol Support | @dprotaso |
+| Percentage-based request mirroring | @jakebennert |
+| Backend TLS Config improvments | @mkosieradzki, @LiorLieberman |
```

Comment:
typo: improvments -> improvements

Reply:
In retrospect I dropped the commit for the changelog, we should follow up over here instead:

https://github.com/kubernetes-sigs/gateway-api/pull/3327

Sorry for any confusion!